crypto/bls/signature_batch.go

```diff
@@ -25,6 +32,11 @@ func (s *SignatureBatch) Join(set *SignatureBatch) *SignatureBatch {
 	s.Signatures = append(s.Signatures, set.Signatures...)
 	s.PublicKeys = append(s.PublicKeys, set.PublicKeys...)
 	s.Messages = append(s.Messages, set.Messages...)
+	if s.Descriptions != nil && set.Descriptions != nil {
```

Comment:
Remove this nil check, since all signature sets now have a `description` attached to it.